.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,8 +11,23 @@ Closes: # (issue)
 # Checklist:
 
 - [ ] I have performed a self-review of my own code
+- [ ] I have added a short summary of my change to the CHANGELOG in website/CHANGELOG.md
 - [ ] My code follows the style guidelines of this project
 - [ ] I have commented my code in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
+
+
+# A note about the CHANGELOG
+
+Hello 👋 and thank you for contributing to Gradio!
+
+All pull requests must update the change log located in website/CHANGELOG.md unless they are labeled with the no-changelog-update label.
+
+Please add a brief summary of the change to the Full Changelog section of the UPCOMING.md file and include the pr number.
+For example, "* Added a cool new feature :pr:123". 
+
+If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.
+If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file directly on the 
+Github web UI since that lets you upload files directly via drag-and-drop.
```

Comment:
```suggestion
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.
```